src/data-table/column-categorical.js

```diff
@@ -28,8 +28,9 @@ type OptionsT = {|
 |};
 
 type FilterParametersT = {|
-  selection: Set<string>,
+  description: string,
```

Comment:
why do we need the additional `description` param?

Reply:
it comes from the complexity around supporting 'custom' columns. How can we translate a custom set of filter properties into a string that is rendered into a 'tag' component